Sources/StreamVideo/WebRTC/PeerConnection.swift

```diff
@@ -115,7 +115,7 @@ class PeerConnection: NSObject, RTCPeerConnectionDelegate, @unchecked Sendable {
     }
     
     func setRemoteDescription(_ sdp: String, type: RTCSdpType) async throws {
-        guard pc.remoteDescription?.sdp != sdp || pc.remoteDescription?.type != type else { return }
+//        guard pc.remoteDescription?.sdp != sdp || pc.remoteDescription?.type != type else { return }
```

Comment:
better to delete this altogether, it's a bit confusing to have commented out code.